extensions/bools.py

```
@@ -1,23 +1,18 @@
-#   INITIALIZATION
-#
-
-# INITIALIZATION -> FUNCTION
-def init() -> None: pass
\ No newline at end of file
+#
+#   BOOLS
+#
+
+# BOOLS -> NAMESPACE
+bools = ContextVar("bools")
```

Ruff (0.11.9)

Undefined name `ContextVar`

(F821)